src/client/java/teammates/client/scripts/helper/DataMigrationForSanitizedDataHelper.java

```diff
@@ -0,0 +1,41 @@
+package teammates.client.scripts.helper;
+
+/**
+ * Utility class that provides methods to help with migrating data to its desanitized form.
+ * Contains a private constructor to prevent instantiation of the class.
```

Comment:
PMD issues a rule violation for not having a private constructor.
```
All methods are static.  Consider using a utility class instead. Alternatively, you could add a private constructor or make the class abstract to silence this warning.
```
(A public empty constructor is not allowed too.)